test/kitchen/site-cookbooks/dd-security-agent-check/recipes/default.rb

```diff
@@ -8,16 +8,11 @@
 if node['platform_family'] != 'windows'
   wrk_dir = '/tmp/security-agent'
 
-  remote_directory wrk_dir do
-    cookbook 'dd-system-probe-check'
-    source 'tests'
-    mode '755'
-    files_mode '755'
+  remote_directory "#{wrk_dir}/ebpf_bytecode" do
+    source 'ebpf_bytecode'
     sensitive true
-    case
-    when !platform?('windows')
```

Comment:
I don't get why the windows platform case changed here?

Reply:
because the whole block is already under an if not windows